pkg/cmd/roachtest/tests/perturbation/index_backfill.go

```diff
@@ -27,7 +28,9 @@ type backfill struct{}
 var _ perturbation = backfill{}
 
 func (b backfill) setup() variations {
-	v := setup(b, 40.0)
+	// TODO(baptist): Track down why this test causes stalls and drop the value
+	// to something more reasonable (like 5) once this is done.
```

Comment:
Do we have an issue we can reference here?

Reply:
Good idea. I created #139262.